Client/Frontend/Browser/TabManager.swift

```diff
@@ -159,6 +168,45 @@ class TabManager : NSObject {
 
         return nil
     }
+    
+    weak var appStateDelegate: AppStateDelegate?
+    
+    private func updateAppState() {
```

Comment:
Nit: I think we can name this method. It changes the app state to be the tab tray. Maybe `setAppStateForTabTray` or something?